lib/index.js

```diff
@@ -1,12 +1,21 @@
-var proj4 = require('./core');
+import proj4 from './core';
+import Proj from "./Proj";
+import Point from "./Point";
+import common from "./common/toPoint";
+import defs from "./defs";
+import transform from "./transform";
+import mgrs from "mgrs";
+import version from "./version";
+import includedProjections from "./includedProjections";
+
 proj4.defaultDatum = 'WGS84'; //default datum
-proj4.Proj = require('./Proj');
+proj4.Proj = Proj;
 proj4.WGS84 = new proj4.Proj('WGS84');
-proj4.Point = require('./Point');
-proj4.toPoint = require('./common/toPoint');
-proj4.defs = require('./defs');
-proj4.transform = require('./transform');
-proj4.mgrs = require('mgrs');
-proj4.version = require('./version');
-require('./includedProjections')(proj4);
-module.exports = proj4;
+proj4.Point = Point;
+proj4.toPoint = common;
+proj4.defs = defs;
+proj4.transform = transform;
+proj4.mgrs = mgrs;
+proj4.version = version;
+includedProjections(proj4);
+export default proj4;
```

Comment:
doing it like this means that anyone importing proj4 looses all the of the benefits of it, of course not doing this breaks compatibility